Makefile

```diff
@@ -1,6 +1,6 @@
 NAME = oscap-anaconda-addon
 
-VERSION = 0.35.0
```

Comment:
The versioning is tricky.
1.x are RHEL8-compatible versions
2.x are RHEL9-compatible versions, and so on.
In case of Fedora, a scheme 0.x.y is used, where x is the respective Fedora version. OTOH, as the addon is not used on Fedora ATM, the likelihood of making a Fedora upstream release is relatively low.

Reply:
Ok, will do.